api_spec/edge_api.json

```diff
@@ -0,0 +1,205 @@
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "array",
+                                    "items": {
+                                        "$ref": "#/components/schemas/Flag"
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "components": {
+        "securitySchemes": {
+            "EnvironmentKeyAuth": { "type": "apiKey", "in": "header", "name": "X-Environment-Key" }
+        },
+        "schemas": {
+            "Flag": {
+                "required": [],
+                "type": "object",
+                "properties": {
+                    "feature": {
+                        "$ref": "#/components/schemas/Feature"
+                    },
+                    "feature_state_value": {
+                        "$ref": "#/components/schemas/FeatureStateValue"
+                    },
+                    "enabled": {
+                        "title": "Enabled",
+                        "type": "boolean"
+                    }
+                }
+            },
+            "FeatureStateValue": {
+                "type": "object",
+                "description": "Can be any of the following types: string, integer, boolean, float"
+            },
+            "Feature": {
+                "title": "Feature",
+                "required": ["name"],
+                "type": "object",
+                "properties": {
+                    "id": {
+                        "title": "ID",
+                        "type": "integer",
+                        "readOnly": true
+                    },
+                    "name": {
+                        "title": "Name",
+                        "type": "string",
+                        "maxLength": 2000,
+                        "minLength": 1
+                    },
+                    "type": {
+                        "title": "Type",
+                        "type": "string",
+                        "maxLength": 50
+                    }
+                }
+            },
+            "createBodyRequest": {
+                "title": "object",
+                "required": true,
+                "type": "object",
+                "properties": {
+                    "Identifier": {
+                        "title": "identifier",
+                        "type": "string",
+                        "maxLength": 50
+                    },
+                    "Traits": {
+                        "title": "Traits",
+                        "type": "object",
+                        "properties": {
+                            "Trait key": {
+                                "title": "trait_key",
+                                "type": "string",
+                                "maxLength": 50
```

Comment:
I don't think this `maxLength` here is correct. There probably isn't a max length on these to be honest. You can just leave this attribute off.